Index: grub-customizer-5.2.5/misc/grub-customizer.desktop
===================================================================
--- grub-customizer-5.2.5.orig/misc/grub-customizer.desktop
+++ grub-customizer-5.2.5/misc/grub-customizer.desktop
@@ -1,4 +1,5 @@
 [Desktop Entry]
+GenericName=Grub Customizer
 Name=Grub Customizer
 Comment=Customize the bootloader (GRUB2 or BURG)
 Exec=grub-customizer
@@ -6,8 +7,7 @@ Terminal=false
 X-MultipleArgs=false
 Type=Application
 StartupNotify=false
-Categories=System;Settings;
+Categories=System;Filesystem;
 Icon=grub-customizer
 X-Ubuntu-Gettext-Domain=grub-customizer
-X-KDE-SubstituteUID=true
 Keywords=GRUB;GRUB2;BURG;configuration;settings;menu;
